scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
++ agreed, it was moved here because of an earlier review comment. I'll move it to `./src/jest`.